### PR TITLE
Khwe 91 create loader

### PR DESCRIPTION
**Pull request by WOLFRIEND**

### CODE REVIEW

File: src/services/__mocks__/FirebaseService.js [hunk 1]
```diff
@@ -1,4 +0,0 @@
-export const FirebaseService = {
-  database() {},
```
Comment (ihor4568) [inline src/services/__mocks__/FirebaseService.js]: why did you remove this file?

File: src/store/actionTypes.js [hunk 2]
```diff
@@ -39,10 +39,6 @@ export const REMOVE_USER_SONG_START = "REMOVE_USER_SONG_START";
 export const REMOVE_USER_SONG_SUCCESS = "REMOVE_USER_SONG_SUCCESS";
 export const REMOVE_USER_SONG_FAIL = "REMOVE_USER_SONG_FAIL";
 
-export const ADD_SONG_START = "ADD_SONG_START";
```
Comment (ihor4568) [inline src/store/actionTypes.js]: why did you remove this actions?

File: src/store/reducers/loader.js [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+    case actionTypes.FETCH_USER_SONGS_START:
+    case actionTypes.FETCH_USER_SONGS_FAIL:
+    case actionTypes.ADD_ALBUMS_START:
+    case actionTypes.ADD_ALBUMS_FAIL:
+    case actionTypes.ADD_ARTISTS_START:
+    case actionTypes.ADD_ARTISTS_FAIL:
+    case actionTypes.FETCH_ALBUM_SONGS_START:
+    case actionTypes.FETCH_ALBUM_SONGS_FAIL:
+    case actionTypes.FETCH_ARTISTS_SONGS_START:
+    case actionTypes.FETCH_ARTISTS_SONGS_FAIL:
```
Comment (ihor4568) [inline src/store/reducers/loader.js]: actually we need to hide loader when FAIL action happens

Comment (WOLFRIEND) [reply src/store/reducers/loader.js]: Fixed
